src/JuMP.jl

```diff
@@ -474,6 +478,40 @@ end
 getValue(arr::Array{QuadExpr}) = map(getValue, arr)
 
 ##########################################################################
+# Norm
+# Container for √(∑ expr)
+type Norm{T}
```

Comment:
Actually, can we make this parametric on the expression:
```
type Norm{T,E<:GenericAffExpr}
    terms::Vector{E}
end
```
So I can use it for JuMPeR?